applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -542,3 +558,12 @@ def _ConstructSolver(self, builder_and_solver, strategy_type):
             raise RuntimeError(f"Undefined strategy type '{strategy_type}'")
 
         return solving_strategy
+
+    def _GetResidualCriterion(self):
```

Comment:
I've appreciated it very much that much that you have changed `_GetTemperatureCriterion` to `_MakeTemperatureCriterion`, since it makes it much clearer that it creates an object rather than retrieving one. Could we do the same thing here? So rename `_GetResidualCriterion` to `_MakeResidualCriterion` (since it is a new method as well)?

Reply:
changed for all geomechanics solvers.